src/test/java/org/eclipse/uprotocol/communication/RpcServerExample.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.eclipse.uprotocol.communication;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.Test;
+
+import org.eclipse.uprotocol.v1.UCode;
+import org.eclipse.uprotocol.v1.UMessage;
+import org.eclipse.uprotocol.v1.UUri;
+
+public class RpcServerExample {
+    @Test
+    public void test_HappyPath() {
+        // Create the test transport
+        final TestUTransport transport = new TestUTransport();
+        
+        // The method that we will serve
+        final UUri method = UUri.newBuilder()
+            .setUeId(4).setUeVersionMajor(1).setResourceId(3)
+            .build();
+        
+        // Register a listener to handle incoming client requests for a given method
+        RequestListener listener = new RequestListener(transport) {
+            @Override
+            void onReceive(UMessage message, CompletableFuture<UPayload> responseFuture) {
+                // Do the processing of the request here
+                // ...
+
+                // Generate an empty response and then complete the future so that the interface can 
+                // automatically send back the response
+                // NOTE: To indicate a failure, complete the future exceptionally passing a UStatusException
+                // and the interface will handle propagating the error to the client.
+                responseFuture.complete(UPayload.packToAny(null));
```

Comment:
how can this be a happy path test when you are packing null into the response?
If this turns into an object it is still pretty bad and the null is just scary

Reply:
The purpose of the test is to simulate a service that is not sending a response message. I'll change this to UPayload.getDefaultInstance()